### DISCUSSION TITLE
How to reopen tabs in chrome or vscode?

### DISCUSSION THREAD

**Discussion by oslfmt** [i3/i3#5843] [Q&A]
I've figured out how to restore *layouts* with i3, with the handy `i3-save-tree` command. However, this just opens specific applications.
I'm wondering how I can get google chrome, or, more importantly, vscode, to reopen all the tabs. This is possible in windows, by just clicking on the chrome icon, or the vscode icon--all the tabs from the last session popup.

Is this possible in i3? Something tells me this is not an i3 problem though, since its within a specific application--any pointers on where to go with this?

**Comment by okraits** [comment]
In Firefox, there's a setting to remember and open all tabs from the previous session. I guess chrome should have a similar setting.
In VS Code, there's the setting `"window.restoreWindows": "all",` which should go into your `settings.json`.
